replicate-to-sql-with-generic-events/README.md

```diff
@@ -0,0 +1,216 @@
+---
+code: true
+type: page
+title: Replicate to a sql database using Generic events
+description: How to replicate to a sql database using Generic events
+---
+
+# Replicate to a sql database using Generic events
+
```

Comment:
In what repo should I add this link ?

I added my link in the root README of this repo since none of each individual how-to are linked to each other

Reply:
Look this page: https://docs.kuzzle.io/how-to/1/sync-data-to-another-database/

If I want to see the How To on Github rather than on the documentation, I don't have any link.  
What I suggest is to add a link like `[See this on Github](https://github.com/kuzzleio/kuzzle-how-to/tree/master/massive-data-import)`

Reply:
Alright, will do for every How-to :+1: